aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -164,6 +164,8 @@ For more information, see [Configuration in ASP.NET Core](xref:fundamentals/conf
 
 This section lists host settings that apply to both HTTP and non-HTTP workloads. By default, environment variables used to configure these settings can have a `DOTNET_` or `ASPNETCORE_` prefix.
 
+<!-- In the following sections, two spaces at end of line are used to force line breaks in the rendered page. -->
```

Comment:
Interesting. I didn't know trailing whitespace was significant in markdown.

Reply:
https://github.com/aspnet/AspNetCore.Docs/pull/14161

Reply:
It is ... or we could use a backslash at the end of a line to get a `<br>` in rendered topics.